app/src/test/kotlin/fr/free/nrw/commons/leaderboard/LeaderboardApiTest.java

```diff
@@ -0,0 +1,83 @@
+package fr.free.nrw.commons.leaderboard;
+
+import com.google.gson.Gson;
+import fr.free.nrw.commons.profile.leaderboard.LeaderboardResponse;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Request.Builder;
+import okhttp3.Response;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
```

Comment:
Please add Javadocs for all new classes that you create :) This applies for methods as well, although we tend to allow exceptions for simple methods like getters and setters.